### PR TITLE
[TESTED] added tailor and actress

### PR DESCRIPTION
**Pull request by ChaoticNeutralGnome**

### CODE REVIEW

File: Games/types/Mafia/roles/cards/RoleDisguiser.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const Card = require("../../Card");
+const { PRIORITY_ROLE_LEANER } = require("../../const/Priority");
+
+module.exports = class RoleLearner extends Card {
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/RoleDisguiser.js:4]: RoleDisguiser

File: Games/types/Mafia/roles/cards/TailorSuit.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                action: {
+                    labels: ["giveItem", "suit"],
+                    priority: PRIORITY_ITEM_GIVER_DEFAULT - 1,
+                    run: function () {
+                        this.actor.role.data.suit = this.target;
+                    }
+                }
+            }
+        };
+        this.listeners = {
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/TailorSuit.js:37]: interesting. so the "list of possible suits" would change e.g. when mimic turns blue, blue appears in the setup.

Comment (ChaoticNeutralGnome) [reply Games/types/Mafia/roles/cards/TailorSuit.js:37]: yes

**Comment by pikulet** [comment]
**tailor tests**
- [ ] tailored role appears with suit on death
- [ ] tailored role appears with suit to seer
- [ ] tailored role appears with suit on orc reveals
- [ ] snoop can find suit
- [ ] when a role is changed (e.g. masons remove villager from game), the village suit is gone

**actress tests**
- [ ] able to disguise on death
- [ ] able to disguise to seer
- [ ] able to disguise role on orc reveals
- [ ] when roleblocked, does not disguise as anything (suit needs to drop after 1 night)

**Comment by pikulet** [comment]
- should actress drop suit after 1 night? e.g. actress that is roleblocked should not have suit
- the suit does not cover the person on death. the setup is simple, just 4 villager 1 actress, act like a blue and vote off the actress. it appears as actress.

File: data/roles.js [hunk 3]
```diff
@@ -401,6 +401,17 @@ const roleData = {
...
+        "Tailor": {
+            alignment: "Mafia",
+            description: [
+                "Gives out suits that make the wearer appear to be a different role."
+            ],
+        },
+        "Actress": {
+            alignment: "Mafia",
+            description: [
+                "Learns a targets role and appears to be that role."]
```
Comment (pikulet) [inline data/roles.js:413]: format :dango:

**Comment by pikulet** [comment]
lgtm

**Pull request merged**